lambdas/postcode-lookup/src/main/java/uk/gov/di/ipv/cri/address/api/handler/PostcodeLookupHandler.java

```diff
@@ -72,6 +72,7 @@ public APIGatewayProxyResponseEvent handleRequest(
 
         try {
             SessionItem sessionItem = sessionService.validateSessionId(sessionId);
+            eventProbe.log(Level.INFO, "found session");
```

Comment:
A `verify` statement(s) should be added to the `PostcodeLookupHandlerTest` class to cover this amend

Reply:
this has been added @tal-nagra